plugins/source/stripe/resources/services/accounts/accounts.go

```diff
@@ -0,0 +1,81 @@
+package accounts
+
+import (
+	"context"
+
+	"fmt"
+	"strconv"
+
+	"github.com/cloudquery/cloudquery/plugins/source/stripe/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/stripe/stripe-go/v74"
+)
+
+func Accounts() *schema.Table {
+	return &schema.Table{
+		Name:        "stripe_accounts",
+		Description: `https://stripe.com/docs/api/accounts`,
+		Transform:   transformers.TransformWithStruct(&stripe.Account{}, client.SharedTransformers(transformers.WithSkipFields("APIResource", "ID"))...),
+		Resolver:    fetchAccounts("accounts"),
+
+		Columns: []schema.Column{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("ID"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "created",
+				Type:     schema.TypeTimestamp,
+				Resolver: schema.PathResolver("Created"),
+				CreationOptions: schema.ColumnCreationOptions{
+					IncrementalKey: true,
+				},
+			},
+		},
+		IsIncremental: true,
+
+		Relations: []*schema.Table{
+			Capabilities(),
+		},
+	}
+}
+
+func fetchAccounts(tableName string) schema.TableResolver {
```

Comment:
Im not quite sure I understood why this is a function that returns a function instead of just a function.

Reply:
This is to be able to use the name of the table, to use in state save/load operations. Could also make it a const in the fn as well I guess, since all is generated anyway.

Reply:
I see. yeah let's use `const` please, I think it will be more clear that way.

Reply:
Done